src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
fix it everywhere
```suggestion
```